### ISSUE TITLE
a11y (critical): Shell brand logo links have no alt or other content

### ISSUE THREAD

**Issue by bmtcril** [openedx/frontend-base#314]
The logo links in the shell header and footer have no alt or text content, causing 2 critical a11y violations on every page that uses it. Just needs alt text on the image and a name on the link, I believe.
